src/RoundTabler/PCIScan.java

```diff
@@ -31,7 +31,7 @@ public PCIScan(){
 		this.pMYSQLColumnTypes = "(mediumtext, longtext, text, tinytext, varchar)";
 	}
 
-	public PCIScan(Connection MYSqlDBConnection, string TableName  ){
+	public PCIScan(Connection MYSqlDBConnection, String TableName  ){
 		this();
 		pDBConnection = DBConnection;
 		pTableName = TableName;
```

Comment:
Im trying to comment on 38 but not working...

I don't think String.Empty exists maybe instead

ScanResult = "";

Reply:
Fixed

Reply:
Final changes and then it will be perfect!

So on line 43 and 49, "pMySQLColumnTypes" needs to be updated to "pMYSQLColumnTypes"

Line 57, "public void ScanMYSQLTables() {" needs to be updated to "public void ScanMYSQLTables() throws SQLException {"

Line 60, "cStatement.executeSelect(pMYSQLColumnSelect);" updated to "cStatement.execute(pMYSQLColumnSelect);"

Reply:
fixed